docs/.eleventy.js

```diff
@@ -39,4 +40,19 @@ module.exports = function(eleventyConfig) {
   // Ignore liquid parsing on these files
   eleventyConfig.ignores.add('email/templates/code');
   eleventyConfig.ignores.add('email/templates/examples');
+
+  eleventyConfig.addTransform("htmlmin", function(content, outputPath) {
+    if (outputPath && outputPath.endsWith(".html")) {
+      const shouldMinify = !content.includes("---");
```

Comment:
It is not clear at a glance why we exclude content containing 3 dashes. It might be worth a comment or rename the `shouldMinify` variable to hint in the right direction.

Reply:
That line for sure could use a comment. We exclude `---` because that's what wraps the front matter of the liquid templates. Without this exclusion, the minification is attempted on the front matter which breaks the build.